pkg/kubectl/.import-restrictions

```diff
@@ -138,7 +138,7 @@
                 "k8s.io/kubernetes/pkg/util/node",
                 "k8s.io/kubernetes/pkg/util/nsenter",
                 "k8s.io/kubernetes/pkg/util/parsers",
-                "k8s.io/kubernetes/pkg/util/pointer",
+                "k8s.io/utils/pointer",
```

Comment:
can you please move this down the list after all `k8s.io/kubernetes` references please?

Reply:
sorry, my mistakes.  Done